steps/workforce/allocations.ts

```diff
@@ -30,9 +30,18 @@ export const allocateCase = async (page: Page, crn: string, allocation: Allocati
     await page.locator('button', { hasText: 'Continue' }).click()
 
     // Confirm allocation
-    await expect(page).toHaveTitle(/.*Allocate to practitioner.*/)
+    await expect(page).toHaveTitle(/.*Choose practitioner | Manage a workforce.*/)
     await page.locator('a', { hasText: 'Continue' }).click()
 
+    // Explain the reason for allocation to this Practitioner
+    await expect(page).toHaveTitle(/.*Explain your decision | Manage a workforce.*/)
+    await page.fill(
+        '#evidenceText',
+        `${allocation.staff.firstName} ${allocation.staff.lastName} is allocated on case with ${crn} has the necessary specialized knowledge, skills, experience, and training to manage it.`
+    )
+    await page.click('#isSensitive')
+    await page.locator('button', { hasText: 'Continue' }).click()
```

Comment:
Now that we're filling in the evidence details, should we also verify the new contact in Delius that comes with it?

Should just be a case of adding a check for the new decision evidence contact here: https://github.com/ministryofjustice/hmpps-probation-integration-e2e-tests/blob/PI-1023/tests/workforce-allocations-to-delius/allocate-new-person.spec.ts#L59